.github/workflows/test-only.yml

```diff
@@ -119,9 +119,11 @@ jobs:
           # invert the check (it should be not 0/not running)
           docker ps
           docker logs sig-test
-          docker inspect --format '{{json .State.Running}}' sig-test
-          if [ $? -eq 0 ]
+          test -z "`docker ps|grep sig-test`"
+          if [ $? -ne 0 ]
```

Comment:
https://github.com/dgtlmoon/changedetection.io/pull/1850#issuecomment-1764704028

Reply:
Do you remember my previous SIGTERM commit didn't pass because of this?